packages/vaadin-date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -5,15 +5,20 @@
  */
 import { IronA11yKeysBehavior } from '@polymer/iron-a11y-keys-behavior/iron-a11y-keys-behavior.js';
 import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-resizable-behavior.js';
-import { dateAllowed, dateEquals, extractDateParts, getClosestDate } from './vaadin-date-picker-helper.js';
 import { addListener } from '@polymer/polymer/lib/utils/gestures.js';
 import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { KeyboardMixin } from '@vaadin/component-base/src/keyboard-mixin.js';
+import { DelegateFocusMixin } from '@vaadin/field-base/src/delegate-focus-mixin.js';
```

Comment:
The only part where `focusElement` is used at the moment is this one:

```
      if (this._noInput && this.focusElement) {
        this.focusElement.blur();
      }
```

Should we use `this.inputElement` instead and remove `DelegateFocusMixin ` from `DatePickerMixin`?

Reply:
Let's keep this for now (we need `DelegateFocusMixin` for focus and blur listeners).